integration-libs/cds/tsconfig.lib.json

```diff
@@ -63,6 +67,12 @@
       "@spartacus/cart/wish-list": ["dist/cart/wish-list"],
       "@spartacus/cart/wish-list/root": ["dist/cart/wish-list/root"],
       "@spartacus/core": ["dist/core"],
+      "@spartacus/order/assets": ["dist/order/assets"],
```

Comment:
Please avoid unnecessary change order of paths eg. in tsconfig files, it's not related to pickup-in-store and it's hard to review then.

If you are using some automated tool/configuration and recommend it, please suggest it in a separate ticket / pull request so that all developers in the project have the same rules/configuration.

Reply:
This was indeed auto-generated by Yarn. We don't really have an opinion on it, but when one of the tools did it, assumed it must be necessary.

Reply:
Removed all the changes not related to pickup in store